src/components/Button/__snapshots__/InputButton.test.tsx.snap

```diff
@@ -0,0 +1,22 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`renders correctly with all props 1`] = `
+<input
+  aria-disabled={true}
+  className="govuk-button class1 class2 govuk-button--start"
+  data-module="govuk-button"
+  disabled={true}
+  id="1234"
+  name="Testname"
+  type="submit"
+  value="Testvalue"
+/>
+`;
+
+exports[`renders correctly without optional props 1`] = `
+<input
+  className="govuk-button"
+  data-module="govuk-button"
+  disabled={true}
```

Comment:
Make sure you're checking the snapshots do actually match the behaviour you were expecting.